src/components/Login.js

```diff
@@ -46,7 +46,7 @@ class Login extends Component {
     if (result === undefined) {
       Actions.login();
     } else {
-      Actions.main();
+      Actions.generalInfo();
```

Comment:
This is just general question but, is there a difference regarding performance between doing Actions.sceneName and doing Actions.push and pop?